packages/web-components/src/components/ic-classification-banner/readme.md

```diff
@@ -16,13 +16,6 @@
 | `upTo`                | `up-to`                | If `true`, "Up to" will be displayed before the classification and country.                            | `boolean`                                                                     | `false`     |
 
 
```

Comment:
I think these css properties still need to stay so that they are displayed on the guidance site